File: Structural-Patterns/Composite/C#/Composite/Implementations/TextNode.cs
public class TextNode : HTMLNode {
    private String text;

    public TextNode(String text){
        this.text = text;
    }

    public void AddChild(HTMLNode node)
    {
        throw new NotImplementedException();
    }

    public String GenerateHtml(int indentLevel) {
        return (new String(' ', indentLevel) + text);
    }
}

File: Structural-Patterns/Composite/Java/Implementations/TextNode.java
package Implementations;

import Interfaces.HTMLNode;

public class TextNode implements HTMLNode {
    private String text;

    public TextNode(String text){
        this.text = text;
    }

    @Override
    public String generateHtml(int indentLevel) {
        return " ".repeat(indentLevel) + text;
    }

    
}
